Example/Pods/Pods.xcodeproj/project.xcworkspace/xcshareddata/IDEWorkspaceChecks.plist

```diff
@@ -0,0 +1,8 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Hmm... should we check in this file change?

Reply:
Thanks @supperspidey! This is safe to check in as it is automatically generated by `cocoapods`. I think the reason we are seeing this diff is that `Uniform`was previously using a pretty outdated version of cocoapods (1.2.0).